data/models/global_roles.py

```diff
@@ -47,6 +47,21 @@ class Meta:
 
 
 class ControlRole(BaseGlobalRole):
+    always_persist = models.BooleanField(
+        default=False,
+        verbose_name="toujours maintenir le rôle",
+        help_text="Ce rôle ne sera pas retiré lors des synchronisations automatiques avec Grist",
```

Comment:
```suggestion
        help_text="Ce rôle ne sera pas retiré lors des synchronisations automatiques",
```
pinaille : c'est la commande qui fait les changements, p-e enlever la mentionne de grist si jamais on l'utilise pas ou on utilise qqch d'autre

Reply:
Fix ici : https://github.com/betagouv/complements-alimentaires/pull/2414/commits/420ba62c22758b71b3d9d5ccc17ccc72b0b67356